.travis.yml

```diff
@@ -1,39 +1,39 @@
 sudo: required
 dist: trusty
-
 services:
   - docker
 
 env:
   global:
-    - DOCKER_VERSION=1.10.3-0~trusty
+  - DOCKER_VERSION=1.10.3-0~trusty
+  - TAG=temp_tag
 
 before_install:
   # List available docker versions.
   - apt-cache madison docker-engine
-  # Update Docker to needed version https://graysonkoonce.com/managing-docker-and-docker-compose-versions-on-travis-ci/.
-  - sudo apt-get -o Dpkg::Options::="--force-confnew" install -y docker-engine=${DOCKER_VERSION}
+  # Update Docker to needed version
+  # https:/graysonkoonce.com/managing-docker-and-docker-compose-versions-on-travis-ci/
+  - sudo apt-get -o Dpkg::Options::="--force-confnew" install -y "docker-engine=${DOCKER_VERSION}"
 
 matrix:
   include:
-    - env:
-      - TAG=inspirehep/python-base:2.7
-      - DOCKERFILE=python_base/Dockerfile 
-      - ARGS='--build-arg INSPIRE_PYTHON_VERSION=2.7'
-    - env:
-      - TAG=inspirehep/python-base:latest
-      - DOCKERFILE=python_base/Dockerfile 
-      - ARGS='--build-arg INSPIRE_PYTHON_VERSION=2.7'
-    - env:
-      - ARGS=''
-      - TAG=inspirehep/elasticsearch:latest
-      - DOCKERFILE=elasticsearch/Dockerfile 
- 
-script:
-  - travis_retry docker build -f $DOCKERFILE $ARGS -t $TAG .
+  - env:
+    - DOCKER_PROJECT=inspirehep/python-base
+    - DOCKER_IMAGE_TAG=2.7
+    - DOCKERFILE=python_base/Dockerfile
+    - ARGS='--build-arg=INSPIRE_PYTHON_VERSION=2.7'
+  - env:
+    - DOCKER_PROJECT=inspirehep/python-base
+    - DOCKER_IMAGE_TAG=latest
+    - DOCKERFILE=python_base/Dockerfile
+    - ARGS='--build-arg=INSPIRE_PYTHON_VERSION=2.7'
+  - env:
+    - DOCKER_PROJECT=inspirehep/elasticsearch
+    - DOCKER_IMAGE_TAG=latest
+    - DOCKERFILE=elasticsearch/Dockerfile
+    - ARGS=''
 
-deploy:
-  provider: script
-  script: ./deploy.sh
-  on:
-    branch: master
+script:
+  - travis_retry docker build -f "$DOCKERFILE" $ARGS -t "$TAG" .
+  - chmod +x deploy.sh
+  - ./deploy.sh
```

Comment:
Why is this in the script and not a `deploy` tag? Maybe it can be done as an `after_success` script.